### DISCUSSION TITLE
Post (and other methods) routes?

### DISCUSSION THREAD

**Discussion by kaleidawave** [vikejs/vike#1144] [Discussion] [answered]
### Description

Does vike support other types of HTTP request?

I seee `.pages.server.js`. Is it just for returning HTML views under the `Page` export?

Could there be a `.post.server.js` which exports a `Post` function which handles http post requests?

Haven't looked into vike too much. But if it could then compile to the [cloudflare pages functions API](https://developers.cloudflare.com/pages/platform/functions/api-reference/) it would exactly what I am looking for!

**Comment by brillout** [comment, marked as answer]
https://brillout.github.io/rules/#try &mdash; that said, I'm happy to assist once you've digged into Vike a little more. Feel free to keep me updated.
